java/memory/memory-netty/src/test/java/org/apache/arrow/memory/TestBaseAllocator.java

```diff
@@ -438,6 +441,99 @@ public ArrowBuf empty() {
         }).build());
   }
 
+  @Test
+  public void testAllocationManagerRequestedSizeAndGrantedSize1() {
```

Comment:
please renamed tests to not be numbered and instead describe what they are testing.

Reply:
done